src/main/scala/com/microsoft/hyperspace/index/rules/RuleUtils.scala

```diff
@@ -79,8 +79,21 @@ object RuleUtils {
       //  See https://github.com/microsoft/hyperspace/issues/158
 
       // Find the number of common files between the source relations & index source files.
-      val commonCnt = inputSourceFiles.count(entry.sourceFileInfoSet.contains)
+      val (commonCnt, commonBytes) = inputSourceFiles.foldLeft(0L, 0L) { (res, f) =>
```

Comment:
nit: Update the comment as the code now computes both common file count and bytes.